doc/features/tls/README.md

```diff
@@ -56,7 +67,9 @@ var cluster = Cluster.Builder()
     .Build();
 ```
 
-### Client authentication
+The examples on the [driver's repository][Github repository] have a basic custom certificate validator which can be used as a starting point.
+
+### Enabling client authentication
 
 To enable client authentication, you need to provide the driver the client certificate(s):
```

Comment:
Unrelated but you might want to fix the grammar: `provide the driver with the client certificate(s)`.

Reply:
nice catch, fixed 👍